application/features/bootstrap/MfaContext.php

```diff
@@ -262,14 +262,6 @@ public function iRequestToVerifyTheWebauthnMfaRegistrationWithALabelOf($label)
         $this->iRequestTheResourceBe('/mfa/' . $this->mfa->id . '/verify/registration', 'created');
     }
 
-    /**
```

Comment:
Why was this removed?

Reply:
I was trying to write a test that would use this, but failed. In the process, I noticed that it was not being used.